src/main/java/frc/robot/commands/operatorcommands/TeleClimbTilt.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands.operatorcommands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.climber.Climber;
+import frc.robot.subsystems.flywheelvarhood.VarFlyWheel;
+
+public class TeleClimbTilt extends CommandBase {
+
+    // r 16
+
+    private final Climber climber;
+
+    public TeleClimbTilt(Climber climber) {
+        this.climber = climber;
+    }
+
+    @Override
+    public void initialize() {
```

Comment:
```suggestion
    public void execute() {
```

Reply:
We should be doing all command activities in the execute method. Initialize should do just that... Initialize anything that needs to happen before the command is executed. Then `return true` in `isFinished()`